source/base/mpi.cc

```diff
@@ -299,7 +299,8 @@ namespace Utilities
       const MPI_Comm &                 mpi_comm,
       const std::vector<unsigned int> &destinations)
     {
-      const unsigned int myid    = Utilities::MPI::this_mpi_process(mpi_comm);
+      const unsigned int myid = Utilities::MPI::this_mpi_process(mpi_comm);
+      (void)myid;
```

Comment:
If `myid` is only used in assertions, then just inline the call to `this_mpi_process` into the assertion.

Reply:
Done.